src/router.py

```diff
@@ -0,0 +1,159 @@
+from router_leg import RouterLeg    
+import pascy
+import socket
+import select
+from functools import lru_cache
+
+ROUTER_LEGS = []
+
+ARP_TABLE = {'1.1.1.2':'02:42:01:01:01:02',
+             '2.2.2.2':'02:42:02:02:02:02'}
+
+BROADCAST = 'FF:FF:FF:FF:FF:FF'
+
+def get_packet(leg : RouterLeg) -> pascy.Layer:
+    # Recieve packet
+    raw_socket = leg.raw_socket
+    pkt_raw = raw_socket.recvfrom(65536)[0]
```

Comment:
What is `65536`?

Reply:
Buffer I can read

Reply:
`65536` is a number, not a buffer...
Please explain yourself more clearly.

Also, make this a constant 🙃 
It's nice to have a file with all constant (usually named `const.py`), as this makes it easier to modify all constants throughout the project.
This is especially useful if you use constants in many places.